examples/07_join_aggregation.py

```diff
@@ -0,0 +1,438 @@
+"""
+TODO : Add some docstring
+----
+"""
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Movies recommendation with explicit labels
+
+# ## Loading data
+
+# In[1]:
+
+
+from skrub.datasets import fetch_movielens
+
+ratings = fetch_movielens(dataset_id="ratings")
+ratings = ratings.X
+ratings
+
+
+# In[2]:
+
+
+import pandas as pd
+
+ratings["timestamp"] = pd.to_datetime(ratings["timestamp"], unit="s")
+ratings["year"] = ratings["timestamp"].dt.year
+ratings["day_name"] = ratings["timestamp"].dt.day_name()
+
+
+# ## Quick Data Exploration
+
+# In[3]:
+
+
+from matplotlib import pyplot as plt
+import seaborn as sns
+
+sns.set_style("darkgrid")
+
+
+def make_barplot(x, y, title):
+    norm = plt.Normalize(y.min(), y.max())
+    cmap = plt.get_cmap("magma")
+
+    ax = sns.barplot(
+        x=x,
+        y=y,
+        palette=cmap(norm(y)),
+    )
+    plt.xticks(rotation=30)
+    plt.xlabel(None)
+    plt.ylabel(None)
+    plt.tight_layout()
+    plt.title(title)
+
```

Comment:
I think you need to add this so that the plot gets rendered in the example
```suggestion
    plt.show()
```